app/.env.example

```diff
@@ -0,0 +1,3 @@
+REACT_APP_CLIENT_ID=672610100992-9ob4krjvvh9kfs8sr0jb4d7hlsvcmcv8.apps.googleusercontent.com
+REACT_APP_API_KEY=AIzaSyA4JVs7ikQGD4LGS49guiAQ6uujfqZFqXw
+REACT_APP_SCOPES=https://www.googleapis.com/auth/calendar.readonly
```

Comment:
I know you already use the new method with George, but just since it's already out here in the open, following the instruction from[ Git Guardian's check](https://github.com/sunnybrie/Timedown/pull/6/checks?check_run_id=2528932617) would be highly advisable. You could rewrite the git history as they suggest in #2 but either way you'd need to revoke the key and generate a new one

Reply:
Generally in an `.env.example` any non-default values scrubbed out, so it becomes a template instead. If someone wants to test it, when they copy the file, they can then get + set the values as needed. 
```
REACT_APP_API_KEY=api_key_here
```